packages/access-client/src/agent-use-cases.js

```diff
@@ -92,20 +92,37 @@ export async function addProvider({ access, space, account, provider }) {
 }
 
 /**
+ * @typedef {(delegations: Ucanto.Delegation<Ucanto.Capabilities>[]) => boolean} DelegationsChecker
+ */
+
+/**
+ * @type DelegationsChecker
+ */
+export function delegationsIncludeSessionProof(delegations) {
```

Comment:
this is a one liner, do we really need to export it for others?
(might it be fine to export just `isSessionProof`? or neither?)

Reply:
I'd like to give people at least one `DelegationsChecker` implementation that they can use yep - more about providing a publicly exported example of a `DelegationsChecker` so they can implement their own version of `waitForAuthorizationByPolling`. 

I'd be down to hide this, but in that case I don't think it's appropriate to take a `DelegationsChecker` in the publicly exported `waitForAuthorizationByPolling` the way we discussed - I'd want to remove the first parameter there and always pass it `delegationsIncludeSessionProof` - honestly I think this is an inferior option but I'm not strongly against it.

Reply:
> I'd like to give people at least one DelegationsChecker implementation that they can use

sgtm